invenio_oauthclient/contrib/cern.py

```diff
@@ -209,12 +209,19 @@ def get_resource(remote):
 def account_info(remote, resp):
     """Retrieve remote account information used to find local user."""
     res = get_resource(remote)
+    session.pop('cern_resource')
 
     email = res['EmailAddress'][0]
     external_id = res['PersonID'][0]
     nice = res['CommonName'][0]
     name = res['DisplayName'][0]
 
+    groups = fetch_groups(res['Group'])
```

Comment:
but if it happens in ```account_setup``` it will only be added to the sessions when you register.. if you go and log in from a different browser it won't be added